test/torrent-destroy.js

```diff
@@ -22,3 +24,34 @@ test('torrent.destroy: destroy and remove torrent', function (t) {
     client.destroy(function (err) { t.error(err, 'client destroyed') })
   })
 })
+
+test('torrent.destroy: seed torrent and remove it', function (t) {
+  t.plan(7)
+
+  var client = new WebTorrent({ dht: false, tracker: false })
+
+  client.on('error', function (err) { t.fail(err) })
+  client.on('warning', function (err) { t.fail(err) })
+
+  client.seed(fixtures.leaves.content, {
+    name: 'Leaves of Grass by Walt Whitman.epub',
+    announce: []
+  }, function (torrent) {
+    t.equal(client.torrents.length, 1)
+    t.equal(torrent.infoHash, fixtures.leaves.parsedTorrent.infoHash)
+    t.equal(torrent.magnetURI, fixtures.leaves.magnetURI)
+
+    var completeFileName = path.join(torrent.path, torrent.files[0].name)
+
+    client.remove(torrent, {'remove': true}, function (err) {
+      t.error(err, 'torrent removed')
+      fs.stat(completeFileName, function (err) {
```

Comment:
I know this might not work well with browser API, I might add a check on the type of `store` used.